### DISCUSSION TITLE
display: none; not work ( model still displaying after other model select in example page)

### DISCUSSION THREAD

**Discussion by chickei** [google/model-viewer#3776] [Q&A]
Hi All, 

I had tried to follow the example in  page in Model Viewer WebXR Demo, link at below:  https://modelviewer.dev/examples/augmentedreality/index.html

But i found that the previous model didn't hide still displaying after i clicked to change other model. Please read the below image i captured from screen. It is the problem about display: none?
  
 /* This keeps child nodes hidden while the element loads */
  :not(:defined) > * {
    display: none;
  }

![image](https://user-images.githubusercontent.com/46594813/188308320-f8485703-22bb-4469-afd3-afe154e0cc2a.png)

Hope anyone can give me some advices how to solve the hide/ show 3 models feature for my project. 

Thanks

**Comment by elalish** [comment]
Can you try to fix the formatting and remove everything that's not required to repro the problem? Also, if you can remix our Glitch (in the issue template) and link to that, it would help a lot with understanding what your repro.

**Reply by elalish** [reply]
Nevermind! This is #3780, which I definitely need to fix. But I already fixed the fact that it was accidentally published early.

**Reply by chickei** [reply]
Hi Elaslish, 
Thank you for your reply and glad to know you are following this isusse.

For my case, actually, i just try to copy the script shown in example (only replace my 3D model), and it had been happened the overlapped models when change different models. 

I had been removed those irrelated script in my discussion already, hope and waiting any news for fixed the overlapped models issues, since it really help me alot for the project that i am doing. 

Thanks Elalish
